.github/workflows/release.yml

```diff
@@ -52,6 +52,7 @@ jobs:
   homebrew:
     name: "Bump Homebrew formula"
     runs-on: ubuntu-22.04
+    needs: goreleaser
```

Comment:
you dont need goreleaser step, it can run parallel.

Reply:
How will it know the version and the sha to bump if this step runs before goreleaser finishes?

Reply:
```suggestion
```